doc/source/methods/ProtoSelect.ipynb

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
<ol><li>Protoselect -&gt; ProtoSelect (check spelling consistent across materials).</li><li>Perhaps we should stress at the beginning that the whole discussion and method applies to classification problems.</li><li>Nitpick - can we have an "argmin" macro with the index l centered below? Not sure if possible in Jupyter Latex... (https://tex.stackexchange.com/questions/5223/command-for-argmin-or-argmax)</li><li>Please link to the Arxiv landing page for the paper, not directly to the pdf (see below too)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>